src/node_dotenv.cc

```diff
@@ -216,7 +224,7 @@ void Dotenv::ParseContent(const std::string_view input) {
         if (newline != std::string_view::npos) {
           value = content.substr(0, newline);
           store_.insert_or_assign(std::string(key), value);
-          content.remove_prefix(newline);
+          content.remove_prefix(newline + 1);
```

Comment:
Why do we make this change? Can you add a comment

Reply:
Previously, we used `content.remove_prefix(newline)`, which would leave the `'\n'` intact. On the next iteration, `content.front()` might be `'\n'`, potentially causing a re-check of the same line or treating it as an empty line.